package.json

```diff
@@ -19,7 +19,7 @@
     "collectCoverage": false
   },
   "dependencies": {
-    "@mapbox/mapbox-gl-style-spec": "^13.23.0",
+    "@mapbox/mapbox-gl-style-spec": "^13.26.0",
```

Comment:
This change isn't directly related to using node 18, but I figured I'd update it for parity with other tools while I'm here. Other dependencies should get more testing.